demo-opensuse/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM opensuse
+MAINTAINER Shikher Verma
```

Comment:
MAINTAINER is [deprecated](https://docs.docker.com/engine/reference/builder/#maintainer-deprecated). Could you change to labels instead?